firefox_puppeteer/tests/test_utils.py

```diff
@@ -0,0 +1,69 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from firefox_ui_harness import FirefoxTestCase
+
+
+class TestUtils(FirefoxTestCase):
```

Comment:
Gna, partly scratch my last comment. It should not have been `helper method` but methods in utils.js. Means each of those methods in utils.js should better get its own class due to complexity. Alone for sanitize we need a couple of test methods in the class.